proto/proto/types/account.proto

```diff
@@ -38,11 +38,13 @@ message AccountStorageHeader {
         // The type of the storage slot.
         uint32 slot_type = 2;
 
-        // The commitment (Word) for this storage slot.
+        // The data (Word) for this storage slot.
+        // For value slots (slot_type=0), this is the actual value stored in the slot.
+        // For map slots (slot_type=1), this is the root commitment of the storage map.
```

Comment:
nit: "root commitment" sounds redundant. We could say just "this is the root of the storage map" or "this is a commitment to the storage map" (the former is probably preferred).